translations/clean-up.zh-cn.md

```diff
@@ -0,0 +1,41 @@
+# 清理你的项目
+
+当你完成每个项目后，请删除你的云端资源。
+
+在每个项目的课程中，也许你创建了下列的某些：
+
+* 一个资源组
+* 一个物联网中心
+* 物联网设备注册
+* 一个存储账户
+* 一个函数应用
+* 一个 Azure 地图账户
+* 一个自定义视觉项目
+* 一个 Azure 容器注册表
+* 一个认知服务资源
+
+这些资源中大部分都不收费 - 完全免费或者是免费层。对于规定了付费层的服务，你的使用额度会在免费范围内，或只花费几分钱。
```

Comment:
@Mireille-T  I've done.May you review it again later? 
And you can add comment to anywhere you request for modifing like this by clicking the line number in `Files changed` tab.